.github/workflows/test_conda-build.yml

```
@@ -18,19 +19,27 @@ jobs:
 
       - name: build conda env
         run: |
-            # The appended lines make sure that we actually build with Cylc
-            # too:
-            echo "  - pip" >> conda-evironment.yml
-            echo "  - pip": >> conda-environment.yml
-            echo "    - ." >> conda-environment.yml
-            cat conda-environment.yml
-            conda env create \
-              -f conda-environment.yml
-            . /usr/share/miniconda/etc/profile.d/conda.sh
+          # The appended lines make sure that we actually build with Cylc too:
+          echo "  - pip": >> conda-environment.yml
+          echo "    - ." >> conda-environment.yml
+          cat conda-environment.yml
```

I added a second suggestion, looks like you used my first which was actually incorrect because of what you said in response

```suggestion
          # https://stackoverflow.com/a/67096191/3217306
          conda_env_file="conda-environment.yml"
          echo "  - pip" >> "$conda_env_file"
          echo "  - pip:" >> "$conda_env_file"
          echo "    - ." >> "$conda_env_file"
          cat "$conda_env_file"
```